geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -78,6 +80,29 @@ class AccumuloBlobStore(ds: AccumuloDataStore) {
     }
   }
 
+  def delete(id: String): Unit = {
+    // TODO: Get Authorizations using AuthorizationsProvider interface
+    // https://geomesa.atlassian.net/browse/GEOMESA-986
+    val bd = connector.createBatchDeleter(blobTableName, new Authorizations(), bwc.getMaxWriteThreads, bwc)
+    bd.setRanges(List(new Range(new Text(id))))
+    bd.delete()
+    bd.close()
+    deleteFeature(id)
+  }
+
+  private def deleteFeature(id: String): Unit = {
+    val removalFilter = Filters.ff.id(Filters.ff.featureId(id))
+    val fd = ds.getFeatureWriter(blobFeatureTypeName, removalFilter, Transaction.AUTO_COMMIT)
+    try {
+      fd.remove()
+    } catch {
+      case e: Exception =>
+        logger.error(e.getMessage)
```

Comment:
it would be better to put a meaningful message and pass the exception like:
`logger.error("Couldn't remove feature from blobstore", e)`